src/goto-programs/instrument_preconditions.cpp

```diff
@@ -11,8 +11,6 @@ Date:   September 2017
 
 #include "instrument_preconditions.h"
 
-#include <util/replace_symbol.h>
```

Comment:
It's okay to move this to the `.h`, but if we don't need the new function introduced in the `.h` (see other comment), I would undo all changes to this file and the `.h` just to minimize this PR.

Reply:
This was debugging leftovers. Sorry. Removed it.